### ISSUE TITLE
[post-FxA Upgrade] Ensure LeanPlum push notification opt-in is working

### ISSUE THREAD

**Issue by garvankeeley** [mozilla-mobile/firefox-ios#6189]
**Acceptance Criteria**

* Ensure Leanplum opt-in is working
* Ensure that receiving push notifications when the app is not open is working

**Background**

Marketing currently has a campaign running that will reach users who have not opened the app in 45 days

**Notes**

LeanPlum had code for push notification opt-in that didn't appear useful, since we ask for that as part of Sync sign-in. Perhaps it was for non-Sync users to get push notifications? I have never seen the app doing that in the past. We will want to re-visit this and understand if there is work to do to setup LeanPlum.

https://github.com/mozilla-mobile/firefox-ios/blob/rust-integration/Client/Application/LeanplumIntegration.swift#L308

**Comment by garvankeeley** [comment]
cc @nbhasin2

**Comment by nbhasin2** [comment]
**Another find:** This was defined for with an A/B test variable. Currently no A/B test is running and value of the variable to show this screen is set to false by default. 

Below is how the notification looks like. Also, its a custom template In-app message notification from leanplum. This seems to be mainly for enabling push notification and why user should do it. It it sent from Leanplum to the device.

a) This is done by setting up template in the app see setupCustomTemplates() in LeanplumIntegration.swift 

b) Once the template is setup, you can then go to messages send it. Currently it can be found under name **FxA Prepush v1**

**Leanplum** 
<img width="1188" alt="image" src="https://user-images.githubusercontent.com/8919439/80249549-0d74aa80-8640-11ea-8b29-50ee7e4cea69.png">

**In the app**
![image](https://user-images.githubusercontent.com/8919439/80249306-963f1680-863f-11ea-91b8-6f371178c41b.png)

**Comment by nbhasin2** [comment]
Closing this as we have commented this code for now. This code is very specific to how we ask for users to allow for push notifications and is not necessarily required by FxA. We already prompt users to allow for push right after they sign in to Firefox iOS 

We will revisit this as it is more for asking users to simply allow Notifications that will help FxA sync and send tabs to work properly in general.
